getting-started/eclipselink/docker-compose-bootstrap-db.yml

```diff
@@ -25,7 +25,7 @@ services:
       polaris.persistence.type: eclipse-link
       polaris.persistence.eclipselink.configuration-file: /deployments/config/eclipselink/persistence.xml
     volumes:
-      - ../assets/eclipselink/:/deployments/config/eclipselink
```

Comment:
Let's revert the changes related to the `assets` folder. There's a better solution than playing with the relative paths in #1610 that I think would be best to use instead - but we're slight a slight bit off from merging that PR through just yet.

Reply:
yes, I'm happy if we abandon this and work with what's started in #1610 ... do you think we should update the doco changes in that PR as well?

Reply:
IT does look like the doco change I applied are also in 1610.. so this can be abandoned... thanks!